### ISSUE TITLE
Audio/Video capture and emission

### ISSUE THREAD

**Issue by FilippoPiazza** [plusvic/simplebus2-intercom#6]
Hello, very nice project.
Do you by chance have figured out how the audio is managed on the bus? What about the video?

**Comment by plusvic** [comment]
Audio/Video is transmitted as an analog signal, specifically some kind of [composite video](https://en.wikipedia.org/wiki/Composite_video). I'm not an expert, so I don't really understand the details.

**Comment by FilippoPiazza** [comment]
This is extremely helpful. Do you have any kind of documentation or is this your own conclusion?

**Comment by plusvic** [comment]
It's my own conclusion based on some signals I was able to capture with an oscilloscope. The signals looked like this https://hackaday.com/2018/01/18/know-your-video-waveform/. So, I'm pretty sure its some sort of composite video. I also researched a little bit and learned that there are some ICs specifically designed for decoding these signals ([example](https://www.analog.com/en/products/max9526.html#product-overview)), but that's beyond my expertise level.
